test/run-test-without-telemetry.mjs

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-console */
+import { execa } from 'execa';
+import path from 'path';
+
+// resolved from the root of the project
+const inputDir = path.resolve('./test/fixtures/without-telemetry/input');
+const binPath = path.resolve('./bin/cli.js');
+const execOpts = { cwd: inputDir, stderr: 'inherit' };
+
+console.log('running codemod');
+
+const codemod = execa(binPath, ['app'], execOpts);
+codemod.stdout.pipe(process.stdout);
+await codemod;
+
+console.log('codemod complete');
+
+console.log('comparing results');
+
+try {
+  await execa('diff', ['-rq', './app', '../output/app'], execOpts);
+} catch (e) {
+  console.error('codemod did not run successfully');
+  console.log(e);
+
+  process.exit(1);
+}
+
+console.log('codemod ran successfully! 🎉');
+process.exit(0);
```

Comment:
Does this test have actual assertions? Or is it like a smoke test?

Reply:
so... unfortunatly the git diff didn't quite fiture out that this was a move-file change. I didn't alter the way the test was working, I'm just changing it to `.mjs` to make it work with the new execa version.

but from my understanding, this does have an "assertion". If you look at the bit that is calling diff on two folders: https://github.com/ember-codemods/ember-angle-brackets-codemod/pull/502/files#diff-f28f8437ce5be5f9a77df8786489b7522a29c3377687b4feb491253b7044a4e2R21 you'll see that it's testing the "app" folder against a known output folder. That comes from the fixtures folder: https://github.com/ember-codemods/ember-angle-brackets-codemod/tree/master/test/fixtures

essentially I've seen people setup codemod tests like this before and it's just testing an input folder against an expected output at the end, and if they match it's good 🎉

Reply:
I hope that answers your question 🎉